src/main/java/application/FormController.java

```diff
@@ -90,7 +90,7 @@
     @UserLock
     @UserRestore
         public NewFormResponse newFormResponse(@RequestBody NewSessionRequestBean newSessionBean,
-                                           @CookieValue(Constants.POSTGRES_DJANGO_SESSION_ID) String authToken) throws Exception {
+                                           @CookieValue(name=Constants.POSTGRES_DJANGO_SESSION_ID, required=false) String authToken) throws Exception {
```

Comment:
what's the difference here for when the view is accessed outside the context of mac digest validation? Will it still return an error if no session id is passed in?